packages/wdio-config/src/lib/ConfigParser.ts

```diff
@@ -37,6 +37,7 @@ interface MergeConfig extends Omit<Partial<TestrunnerOptionsWithParameters>, 'sp
 }
 
 export default class ConfigParser {
+    #projectRootDir = process.cwd()
```

Comment:
this seems to be a static value that never changes which means we should not attach it to the class